source/guide_foundryvtt.rst

```diff
@@ -0,0 +1,221 @@
+.. highlight:: console
+
+.. author:: revilowaldow <oliver@warlow.engineer>
+.. author:: Glumbosch <glumbosch.home.blog>
+
+.. tag:: self-hosting
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/foundryvtt.png
+      :align: center
+
+############
+Foundry VTT
+############
+
+.. tag_list::
+
+
+  `Foundry Virtual Tabletop <https://foundryvtt.com/>`_ is an alternative to Roll20 and many other platforms that enable you to play tabletop role playing games online.
+  Foundry is not free software, but you can host it on your own server.
+  
+  The first step is to purchase a license from their `website <https://foundryvtt.com/>`_.
+  
+  If you pay by SEPA transfer the payment can take up to a week, faster payment methods are available.
+
+----
+
+.. note:: This guide assumes you are completely unfamiliar with the basic concepts of linux
```

Comment:
Thank you for your comments, no worries about the delay.

The whole guide is obviously targeted at novice users.
The SSH introduction included in the guide is deliberately brief so that only the information the user needs to get this specific application set-up is included.

Once the application is running the user would likely never need to use the terminal again (except to maybe restart the service).

If at all possible, keeping the SSH guide would be preferable.